ott/core/icnn.py

```diff
@@ -32,9 +32,11 @@
 
 
 class ICNN(nn.Module):
-  """Input convex neural network (ICNN) architeture with initialization.
+  """Input convex neural network (ICNN) architecture with initialization.
 
-  Containing initialization schemes introduced in Bunne+(2022).
+  Implementation of input convex neural networks as introduced in
+  :cite:`amos:17. Containing initialization schemes proposed by
```

Comment:
You're missing the 2nd \` in :cite:\`amos:17\`, causing sphinx to parse the docstring incorrectly as `Implementation of input convex neural networks as introduced in [].`
(note to self - consider treating warnings as errors when building docs).
Furthermore, would rephrase `... . Containing initialization`  to either `, containing initialization schemes` or `... . Contains initialization` (sorry for being picky).
Lastly, the `Bunne+(2022)` below is not really necessary, the :cite:\`bunne:22\` already renders it nicely.

Reply:
Fixed it! Thanks a lot!